src/battle-log.ts

```diff
@@ -291,6 +295,37 @@ class BattleLog {
 		lastNode.appendChild(document.createTextNode(' '));
 		lastNode.appendChild(button);
 	}
+	unlinkChatFrom(userid: ID) {
+		const classStart = 'chat chatmessage-' + userid + ' ';
+		const innerNodeList = this.innerElem.getElementsByTagName('a');
+		const preemptNodeList = this.preemptElem.getElementsByTagName('a');
+
+		const unlinkNodeList = (nodeList: NodeList) => {
```

Comment:
```suggestion
		const unlinkNodeList = (nodeList: NodeList | HTMLCollection) => {
```

In theory, this should be all that's needed here.

Reply:
```suggestion
		const unlinkNodeList = (nodeList: ArrayLike<HTMLAnchorElement>) => {
```

Actually, this might be better. Feel free to just use `ArrayLike` if VS Code doesn't like `<HTMLAnchorElement>`.